spec.html

```diff
@@ -23364,17 +23363,17 @@ <h1>eval ( _x_ )</h1>
         1. Assert: The execution context stack has at least two elements.
         1. Let _callerContext_ be the second to top element of the execution context stack.
         1. Let _callerRealm_ be _callerContext_'s Realm.
-        1. Let _calleeRealm_ be the current Realm Record.
-        1. Perform ? HostEnsureCanCompileStrings(_callerRealm_, _calleeRealm_).
-        1. Return ? PerformEval(_x_, _calleeRealm_, *false*, *false*).
+        1. Return ? PerformEval(_x_, _callerRealm_, *false*, *false*).
       </emu-alg>
 
       <emu-clause id="sec-performeval" aoid="PerformEval">
-        <h1>Runtime Semantics: PerformEval ( _x_, _evalRealm_, _strictCaller_, _direct_ )</h1>
-        <p>The abstract operation PerformEval with arguments _x_, _evalRealm_, _strictCaller_, and _direct_ performs the following steps:</p>
+        <h1>Runtime Semantics: PerformEval ( _x_, _callerRealm_ _strictCaller_, _direct_ )</h1>
```

Comment:
```suggestion
        <h1>Runtime Semantics: PerformEval ( _x_, _callerRealm_, _strictCaller_, _direct_ )</h1>
```

Reply:
[Commit suggestion] didn't work, so did it manually.